packages/next/test/fixtures/00-app-dir/package.json

```
@@ -3,5 +3,8 @@
     "next": "canary",
     "react": "experimental",
     "react-dom": "experimental"
+  },
+  "scripts": {
+    "vercel-build": "next build"
```

```suggestion
    "build": "next build"
```

Lets use the canonical `build`

---

I have no idea why that file is changed lol